plugins/module_utils/helm.py

```diff
@@ -158,3 +159,15 @@ def parse_helm_plugin_list(module, output=None):
         ret.append((name, version, description))
 
     return ret
+
+
+def get_helm_version(module, helm_bin):
+
+    helm_version_command = helm_bin + " version"
+    rc, out, err = module.run_command(helm_version_command)
+    if rc == 0:
+        m = re.match(r'version.BuildInfo{Version:"v([0-9\.]*)",', out)
+        if m:
+            return m.group(1)
+        return m.group(1)
```

Comment:
I think this line should be deleted now?

Reply:
yes thanks!!